arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -557,14 +505,28 @@
     "SubscriptionIdParameter": {
       "name": "subscriptionId",
       "in": "path",
-      "description": "Microsoft Azure subscription id.",
+      "description": "The Microsoft Azure subscription ID.",
       "required": true,
       "type": "string"
     },
     "ApiVersionParameter": {
       "name": "api-version",
       "in": "query",
-      "description": "Client Api Version.",
+      "description": "The client API version.",
+      "required": true,
+      "type": "string"
+    },
+    "ResourceGroupParameter": {

```

Comment:
I don't think this is right, unless I am missing something here. Why are we creating ResourceGroupParameter and RegistryNameParameter as global parameters? They shouldn't be required as parameters to create a management client, otherwise a different client has to be created for each resource group and registry name, let alone several operations don't need resource group or registry name. I suppose you want to create a single definition for resource group and registry name, then putting them in definitions would be the right approach, but I doubt if creating a separate object with one string variable as the only property is necessary.